show/chassis_modules.py

```diff
@@ -0,0 +1,63 @@
+import click
+from natsort import natsorted
+from tabulate import tabulate
+from swsssdk import SonicV2Connector
+
+import utilities_common.cli as clicommon
+
+CHASSIS_MODULE_INFO_TABLE = 'CHASSIS_MODULE_TABLE'
+CHASSIS_MODULE_INFO_KEY_TEMPLATE = 'CHASSIS_MODULE {}'
+CHASSIS_MODULE_INFO_DESC_FIELD = 'desc'
+CHASSIS_MODULE_INFO_SLOT_FIELD = 'slot'
+CHASSIS_MODULE_INFO_OPERSTATUS_FIELD = 'oper_status'
+CHASSIS_MODULE_INFO_ADMINSTATUS_FIELD = 'admin_status'
+
+@click.group(cls=clicommon.AliasedGroup)
+def chassis_modules():
+    """Show chassis-modules information"""
+    pass
+
+@chassis_modules.command()
+@clicommon.pass_db
+@click.argument('chassis_module_name', metavar='<module_name>', required=False)
+def status(db, chassis_module_name):
+    """Show chassis-modules status"""
+
+    header = ['Name', 'Description', 'Slot', 'Oper-Status', 'Admin-Status']
```

Comment:
module/card 'Name' field has logical slot number at the end.
For the end-user/operator usability aspect, can 'Slot' header be changed to 'Phy-Slot' or 'Physical-Slot' or 'BP-Slot'?  This would help provide clear distinction between logical slot and physical slot in show chassis-module output.

admin@sonic:~$ show chassis-modules status
         Name                      Description             Slot    Oper-Status    Admin-Status
-------------  -----------------  ------  -------------  --------------
CONTROL-CARD1           cpm2-ixr                  16         Online              up

Reply:
Changed to 'Physical-Slot'